content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -92,8 +102,8 @@ Kubernetes provides built-in signers that each have a well-known `signerName`:
        but it is often not the only cluster-admin subject in a cluster.
     1. Permitted x509 extensions - honors subjectAltName and key usage extensions and discards other extensions.
     1. Permitted key usages - must include `["client auth"]`. Must not include key usages beyond `["digital signature", "key encipherment", "client auth"]`.
-    1. Expiration/certificate lifetime - minimum of CSR signer or request.
-       The signer is responsible for checking that the certificate lifetime is valid and permissible.
+    1. Expiration/certificate lifetime - set by the cluster signing duration configuration
+       option of the kube-controller-manager.
```

Comment:
the implementation in kube-controller-manager is one implementation of this signer, but it can be replaced by a custom implementation. I would rephrase this as "set by the `--cluster-signing-duration` option for the kube-controller-manager implementation of this signer" (same comment applies to other places)

Reply:
Indeed! I updated the language with your suggestion.